### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Jamulus app](https://hosted.weblate.org/projects/jamulus/jamulus-app/).

It also includes following components:

* [Jamulus/Windows Installer](https://hosted.weblate.org/projects/jamulus/windows-installer/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by henkdegroot** [comment]
Fixes: https://github.com/jamulussoftware/jamulus/issues/3125

**Comment by ann0see** [comment]
All here, please check the diff: https://github.com/jamulussoftware/jamulus/compare/91cee92e0f3022c8bd5e4fb8c3e6457b7a32628c..e3cd3138f8e15171033815cca80d8b898a3c7e5c
if that's still up to date.

**Review by BLumia — DISMISSED** [review]

**Comment by jujudusud** [comment]
There is a problem with French.
All the modifications made last saturday are reverted with thoses commits.

**Review by ignotus666 — APPROVED** [review]

**Comment by ann0see** [comment]
@jujudusud I hope a lot of the stuff is fixed now. Please recheck and fix it on weblate.

**Comment by trebmuh** [comment]
It looks fine for me now.
Note that I didn't review the app translation part yet.

**Comment by jujudusud** [comment]
I'll look at it next Monday.

**Comment by pljones** [comment]
https://github.com/jamulussoftware/jamulus/actions/runs/5755677872/job/15603622494?pr=3119#step:10:403
More "out of heap space"...  Are we taxing the Github build system too hard?

**Comment by trebmuh** [comment]
@ann0see : the weblate is still in a locked state. I'd like to take some time this weekend to progress on translation review if possible. Can you please tell me when it' going to be released for translation? Thanks.

**Comment by ann0see** [comment]
As soon as the two remaining PRs (the macOS one specifically) are merged, I'll unlock it.

**Comment by ann0see** [comment]
@trebmuh it should be unlocked now.

**Comment by trebmuh** [comment]
Thanks for the work and the ping @ann0see . Yesterday, I started translating again.
I'm on a break for 2 days and will resume afterwards.

**Review by jujudusud — APPROVED** [review]

**Review by ann0see — COMMENTED** [review]
Ok some comments. Thanks for all your work!

File: src/translation/translation_nb_NO.ts [hunk 1]
```diff
@@ -1104,12 +1104,12 @@ We muted your channel and activated &apos;Mute Myself&apos;. Please solve the fe
     <message>
         <location filename="../clientsettingsdlg.cpp" line="150"/>
         <source>If the selected sound card device offers more than one input or output channel, the Input Channel Mapping and Output Channel Mapping settings are visible.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Hvis valgt lydkortsenhet tilbyr mer enn én inn- eller utgangskanal, vil inngangskanaltilknytning og utgangskanaltilknytningsinnstillinger bli synlige.</translation>
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply src/translation/translation_nb_NO.ts:1107]: @ann0see I need to check what "mapping" is elsewhere, for consistency.
Also, it should be `"Input Channel mapping" settings`, but really all lowercase makes the most sense.

File: src/translation/translation_fr_FR.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@
     <message>
         <location filename="../util.cpp" line="577"/>
         <source>There is a server which collects  the audio data from each client, mixes the audio data and sends the mix  back to each client.</source>
-        <translation>Un serveur collecte les données audio de chaque client, les mixe et renvoie le résultat à chaque client.</translation>
+        <translation>Il y a un serveur qui collecte  les données audio de chaque client, les mixe et renvoie le résultat  à chaque client.</translation>
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply src/translation/translation_fr_FR.ts:29]: The two double-spaces surely aren't.
It is more of a direct translation, of something that wasn't good to begin with.
"There is" can be omitted.

File: src/translation/translation_nb_NO.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (henkdegroot) [reply src/translation/translation_nb_NO.ts:1107]: > Also, it should be `"Input Channel mapping" settings`,

Not really:
image

As it should match the language/text used in the app.

[the remark this answers is not in the document]
Comment (comradekingu) [reply src/translation/translation_nb_NO.ts:1107]: I don't think title case is correct there, or for the category it lists.
My point is it needs to be quoted in as a direct use.
Italics are also an option.

File: src/translation/translation_fr_FR.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (jujudusud) [reply src/translation/translation_fr_FR.ts:29]: The two spaces after "collects" and after "mix" are present in the English sentence. The translator has therefore followed the original sentence, even if this is not correct in French.

**Comment by jujudusud** [comment]
fixes: #3123

File: src/translation/translation_fr_FR.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (comradekingu) [reply src/translation/translation_fr_FR.ts:29]: Nor is it correct in English. There is a lot of low-hanging fruit to deal with for the next release :)

[the remark this answers is not in the document]
Comment (trebmuh) [reply src/translation/translation_fr_FR.ts:29]: No, it's not correct but it is in the original (English) version in first place.

File: src/translation/translation_nb_NO.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (pljones) [reply src/translation/translation_nb_NO.ts:1107]: It should match the UI so there's no ambiguity about what is being referred to.  In fact, it's probably too brief.  It should make it clear why the `Input Channel Mapping` and `Output Channel Mapping` options are _not_ be displayed - i.e. if there is only one stereo pair available for each.

Also, adding something brief, like "If displayed, the mappings allow you to choose which (mono) input or output is mapped to the Left or Right channel sent to or from the server", could make it clearer, too.

**Comment by pljones** [comment]
Is there a way to see on Weblate whether any translation activity is taking place?

If those translations we're expecting to progress are now considered completion, should we close out the app translation processes?

**Comment by ann0see** [comment]
Yes. I think there's an activity overview. https://hosted.weblate.org/projects/jamulus/#history

**Comment by trebmuh** [comment]
I should have finish to review the French app translations in a few days.

**Comment by pljones** [comment]
> Yes. I think there's an activity overview. https://hosted.weblate.org/projects/jamulus/#history

It's things like this I could do with having added to the Release Process guide around Weblate.  Very useful, thanks!

**Review by trebmuh — APPROVED** [review]
Just checked and fixes a few stuff on weblate.
All will be good after weblate merging from my point of view.

**Comment by ann0see** [comment]
@BLumia there was a further change in the Chinese translation

**Comment by BLumia** [comment]
> @BLumia there was a further change in the Chinese translation

Doesn't seem like there are new untranslated strings right now...

**Comment by ann0see** [comment]
@BLumia yes. There however was a change you didn't do: https://github.com/jamulussoftware/jamulus/pull/3119/commits/b921086ddb0b1eb6ad2ad2d7506dd95fd0b166ca

**Review by ann0see — COMMENTED** [review]
Merged.

**Pull request merged**